src/index.js

```diff
@@ -466,7 +469,12 @@ export default class AuthnWidget {
   }
 
   enableSubmit() {
-    let nodes = (document.querySelectorAll('input.required[type=text]:not(:disabled), input.required[type=password]:not(:disabled), input.required[type=email]:not(:disabled)'));
+    let nodes = [];
+    document.querySelectorAll('input.required[type=text]:not(:disabled), ' +
+      'input.required[type=password]:not(:disabled), input.required[type=email]:not(:disabled)').forEach(ele => nodes.push(ele));
+    if (this.store.getStore().status === 'REGISTRATION_REQUIRED') {
```

Comment:
Does the code after this work for all field types? I'm not entirely sure, but it looks to me like it might have issues with required dropdown fields

Reply:
hmm, you're right. I'll address this.